pinot-broker/src/main/java/org/apache/pinot/broker/queryquota/HelixExternalViewBasedQueryQuotaManager.java

```diff
@@ -165,11 +157,13 @@ private void createOrUpdateRateLimiter(String tableNameWithType, ExternalView br
       QuotaConfig quotaConfig) {
     if (quotaConfig == null || quotaConfig.getMaxQueriesPerSecond() == null) {
       LOGGER.info("No qps config specified for table: {}", tableNameWithType);
+      buildEmptyOrResetRateLimiterInQueryQuotaEntity(tableNameWithType);
       return;
     }
 
     if (brokerResource == null) {
       LOGGER.warn("Failed to init qps quota for table {}. No broker resource connected!", tableNameWithType);
+      buildEmptyOrResetRateLimiterInQueryQuotaEntity(tableNameWithType);
```

Comment:
Do we still want to update it if EV is null (btw, can you change the var name to be `brokerResourceEV` ?)

Reply:
Even if brokerResourceEV is null for now, it can still be not null in the future. In that case, the metrics are still needed to be appeared. Thus, the query quota entity is still needed.

I'll update the variable name in the next push.